scm-step/README.md

```diff
@@ -64,7 +64,19 @@ You may specify `poll: false` to disable polling for an SCM checkout.
 # Built-in integrations
 
 Currently there are special integrations with the Git (`git` step) and Subversion (`svn` step) plugins.
-At the moment these are very simple and take just a `url` parameter.
+
+## git
+
+Git step let you define a repository to clone. For authentication, a reference to a configured credential can
+be passed.
+
+```
+git url: 'git@git.local.domain:ps/project.git', credentialsId:67c3072d-b9a7-44fa-a5aa-560ba9c1662f
```

Comment:
Since you can now pick the ID for a newly created credentials item, and the `git` step is specifically optimized for writing without snippet generation, it would make more sense to show as an example a human-readable ID.